.github/workflows/test.yml

```diff
@@ -0,0 +1,43 @@
+name: Maid
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
I don't seem to be able to get these builds to trigger.  That said, I ran the specs on a GitHub codespace and I can see that the specs pass for at least Ruby 3.1.3.  I'm opting to merge now so that this can move forward.  I hope to make a PR to test whether CI works, but I may leave that to you over on the Maid organization.

```
Finished in 1.19 seconds (files took 0.94519 seconds to load)
179 examples, 0 failures

$ ruby --version
ruby 3.1.3p185 (2022-11-24 revision 1a6b16756e) [x86_64-linux]
```

Reply:
There seems to be a syntax error in the file, which might be why it's not picking the workflow up. I'll look into it now.

Reply:
Do you want a bit more time before I merge?  I was hoping to merge before trying to move this to the organization, just in case it doesn't go as smoothly as I might like.

Reply:
I fixed an indentation issue, maybe it shows in the repo's settings now? Otherwise, merge whenever you're happy with the PR and I'll take a look once I have access via the `maid` org.

Reply:
> I fixed an indentation issue, maybe it shows in the repo's settings now?

I'm not seeing a difference right now.  I'm guessing it's some odd issue because we don't have any actions on `master`, but that's just a guess.